### PR TITLE
Update lspci function output

### PR DESCRIPTION
**Pull request by sbz**
The current lspci display only the bus selector and device name. Add the
description, vendor and device id similar to the output of lspci -nn

### CODE REVIEW

**Comment by sbz** [comment]
Hi @bsdjhb, I came up with this idea

Before it was looking like below:

```
(kgdb) lspci
0xfffff80003acad80: pci0:0:0:0 (hostb0)
0xfffff80003acac00: pci0:0:2:0 (vgapci0)
0xfffff80003acaa80: pci0:0:3:0 (hdac0)
0xfffff80003aca900: pci0:0:20:0 (xhci0)
0xfffff80003aca780: pci0:0:22:0
0xfffff80003aca600: pci0:0:25:0 (em0)
0xfffff80003aca480: pci0:0:27:0 (hdac1)
0xfffff80003aca300: pci0:0:28:0 (pcib1)
0xfffff80003aca180: pci0:0:28:1 (pcib2)
0xfffff80003aca000: pci0:0:29:0 (ehci0)
0xfffff80003ac8d80: pci0:0:31:0 (isab0)
0xfffff80003ac8c00: pci0:0:31:2 (ahci0)
0xfffff80003ac8a80: pci0:0:31:3 (ichsmb0)
0xfffff80003ac8900: pci0:0:31:6 (pchtherm0)
0xfffff80003eabd80: pci0:2:0:0 (rtsx0)
0xfffff80003eabc00: pci0:3:0:0 (rtwn0)
```

After it looks like that:

```
(kgdb) lspci
0xfffff80003acad80: pci0:0:0:0 (hostb0) Host to PCI bridge [8086:1604]
0xfffff80003acac00: pci0:0:2:0 (vgapci0) VGA-compatible display [8086:1616]
0xfffff80003acaa80: pci0:0:3:0 (hdac0) Intel Broadwell HDA Controller [8086:160c]
0xfffff80003aca900: pci0:0:20:0 (xhci0) Broadwell Integrated PCH-LP chipset USB 3.0 controller [8086:9cb1]
0xfffff80003aca780: pci0:0:22:0
0xfffff80003aca600: pci0:0:25:0 (em0) Intel(R) I218-LM (3) [8086:15a2]
0xfffff80003aca480: pci0:0:27:0 (hdac1) Intel Broadwell HDA Controller [8086:9ca0]
0xfffff80003aca300: pci0:0:28:0 (pcib1) ACPI PCI-PCI bridge [8086:9c9a]
0xfffff80003aca180: pci0:0:28:1 (pcib2) ACPI PCI-PCI bridge [8086:9c94]
0xfffff80003aca000: pci0:0:29:0 (ehci0) Intel Wildcat Point-LP USB 2.0 controller [8086:9ca6]
0xfffff80003ac8d80: pci0:0:31:0 (isab0) PCI-ISA bridge [8086:9cc3]
0xfffff80003ac8c00: pci0:0:31:2 (ahci0) Intel Wildcat Point-LP AHCI SATA controller [8086:9c83]
0xfffff80003ac8a80: pci0:0:31:3 (ichsmb0) Intel Wildcat Point-LP SMBus controller [8086:9ca2]
0xfffff80003ac8900: pci0:0:31:6 (pchtherm0) Wildcat Point Thermal Subsystem [8086:9ca4]
0xfffff80003eabd80: pci0:2:0:0 (rtsx0) 2.0c Realtek RTS5227 PCI MMC/SD Card Reader [10ec:5227]
0xfffff80003eabc00: pci0:3:0:0 (rtwn0) Realtek RTL8192EE [10ec:818b]
```

Extracted from a kgdb session on my current laptop.

**Comment by bsdjhb** [comment]
We could always print the vendor and device IDs perhaps even if there's no active device.  Also, would be good to keep the indentation consistent, but looks good to me overall.

File: gdb/gdb6 [hunk 1]
```diff
@@ -1980,7 +1980,9 @@ define lspci
 	printf "%p: pci%d:%d:%d:%d", $_dinfo, $_dinfo->cfg.domain, \
 	    $_dinfo->cfg.bus, $_dinfo->cfg.slot, $_dinfo->cfg.func
 	if ($_dinfo->cfg.dev != 0 && $_dinfo->cfg.dev->nameunit != 0)
-	    printf " (%s)", $_dinfo->cfg.dev->nameunit
+        printf " (%s) %s [%x:%x]", $_dinfo->cfg.dev->nameunit, \
+        $_dinfo->cfg.dev->desc, $_dinfo->cfg.vendor,
+        $_dinfo->cfg.device
 	end
 	printf "\n"
```
Comment (bsdjhb) [inline gdb/gdb6]:
```suggestion
	    printf " (%s) %s", $_dinfo->cfg.dev->nameunit, \
		$_dinfo->cfg.dev->desc
	end
	printf " [%x:%x]\n", $_dinfo->cfg.vendor, $_dinfo->cfg.device
```

**Comment by sbz** [comment]
Updated as you mentioned 👍

**Pull request merged**